Makefile

```diff
@@ -73,14 +75,31 @@ py/janitor/site/_static/pygments.css:
 
 clean:
 
-SHA=$(shell git rev-parse HEAD)
+docker-%:
+	$(MAKE) build-$*
+	$(MAKE) push-$*
 
-docker-%: core
+build-%:
 	buildah build --no-cache -t ghcr.io/jelmer/janitor/$*:$(DOCKER_TAG) -t ghcr.io/jelmer/janitor/$*:$(SHA) -f Dockerfile_$* .
+
+push-%:
 	buildah push ghcr.io/jelmer/janitor/$*:$(DOCKER_TAG)
 	buildah push ghcr.io/jelmer/janitor/$*:$(SHA)
 
-docker-all: docker-site docker-runner docker-publish docker-archive docker-worker docker-git_store docker-bzr_store docker-differ docker-ognibuild_dep
+docker-all:
```

Comment:
Please use dependencies here (with patsubst?) so that parallel builds work.

Reply:
Done... _I think_